modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImplTest.java

```diff
@@ -326,7 +326,19 @@ public void testGetDDMFormFieldValue() throws Exception {
 			value
 		);
 
-		_testGetDDMFormFieldValue(ddmFormFieldValue, 1, value);
+		_testGetDDMFormFieldValue(
+			ddmFormFieldValue, 1, value, DDMFormFieldTypeConstants.TEXT);
+
+		Mockito.when(
+			_ddmFormFieldValueRenderer.render(
+				ddmFormFieldValue, LocaleUtil.BRAZIL)
+		).thenReturn(
+			StringPool.BLANK
+		);
```

Comment:
@vngoliveira since you are mocking the `render` method when the field type is Image, I don't think you are testing this [change](https://github.com/liferay-bpm/liferay-portal/blob/6c9d00b794a63b87339a073599832d4e359c42ec/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/image/ImageDDMFormFieldValueRenderer.java#L39-L42).

- The logic from a method is not executed when you mock it.

You should create a `ImageDDMFormFieldValueRendererTest` 

- See [CheckboxDDMFormFieldValueRendererTest](https://github.com/liferay-bpm/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/checkbox/CheckboxDDMFormFieldValueRendererTest.java).